src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,41 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final int MAX_STORAGE_NUMBER = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = (K[]) new Object[MAX_STORAGE_NUMBER];
+        this.values = (V[]) new Object[MAX_STORAGE_NUMBER];
+    }
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < MAX_STORAGE_NUMBER; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < size; i++) {
```
we can reduce number of iterations performed